composable_kernel/include/utility/conv_utils.hpp

```diff
@@ -0,0 +1,198 @@
+#ifndef CONV_UTILS_HPP

```

Comment:
composable_kernel/composable_kernel/ should be all GPU code.

This file need to move to composable_kernel/device_operation